### ISSUE TITLE
Best Model/Approach for Multilingual Classification?

### ISSUE THREAD

**Issue by ekoenitz** [UKPLab/sentence-transformers#1458]
Hello, I am trying to use this library for a multilingual classification setup, where it can take one sentence of any target language as input, and produces a label as output.

I have followed the [tutorial](https://www.sbert.net/examples/training/cross-encoder/README.html) for the Cross Encoder, using distilroberta-base as the model and setting InputExample.texts to a list of one sentence for each example (as explained [here](https://github.com/UKPLab/sentence-transformers/issues/1328#issuecomment-998796422)).

This works alright, but non-English accuracy is relatively low, and my supervisor suggested it's because distilroberta-base is only trained on English data.  However, looking at the [pre-trained cross-encoder list](https://www.sbert.net/docs/pretrained_cross-encoders.html), I can't find one that is trained on multilingual data.  Further, the [multilingual models](https://www.sbert.net/docs/pretrained_cross-encoders.html) cannot be loaded with the Cross Encoder nor HuggingFace's AutoModelForClassification.

So, I'd like to ask what's the best way to achieve multilingual classification?  Should I [stick with cross-encoders](https://github.com/UKPLab/sentence-transformers/issues/1231), or make the multilingual models compatible with HuggingFace classifiers as the responses to [this issue](https://github.com/UKPLab/sentence-transformers/issues/513) suggest?

**Comment by nreimers** [comment]
Yes, you need to use a multilingual model, for example:
https://huggingface.co/xlm-roberta-base

**Comment by ekoenitz** [comment]
Just what I was looking for, thank you!

**Comment by ekoenitz** [comment]
Actually, is it possible/preferable to use the models you introduced in [this paper](https://arxiv.org/pdf/2004.09813.pdf) with the CrossEncoder?  They look stronger than xlm-roberta-base and you mention being able to use them for classification in [another issue](https://github.com/UKPLab/sentence-transformers/issues/513), but I'm not sure how the fact that they're primarily sentence embedding models interacts with what the documentation says about cross-encoders being stronger for classification.

**Comment by nreimers** [comment]
For classification I would use xlm-roberta-base with CrossEncoder.

The linked paper talks about embeddings, e.g. as used for search

**Comment by ekoenitz** [comment]
Sounds good, thanks for the clarification.
